content/en/apps/tutorials/sms-forms.md

```diff
@@ -0,0 +1,413 @@
+        },
+        "short": {
+          "en": "HH ID"
+        }
+      },
+      "position": 0,
+      "type": "string",
+      "length": [
+        1,
+        13
+      ],
+      "required": true
+    },
+    "gender": {
+      "labels": {
+        "tiny": {
+          "en": "gender"
+        },
+        "description": {
+          "en": "Gender"
+        },
+        "short": {
+          "en": "Gender"
+        }
+      },
+      "position": 1,
+      "type": "string",
+      "length": [
+        1,
+        6
+      ],
+      "required": true
+    },
+    "age": {
+      "labels": {
+        "tiny": {
+          "en": "age"
+        },
+        "description": {
+          "en": "Age"
+        },
+        "short": {
+          "en": "Age"
+        }
+      },
+      "position": 2,
+      "type": "integer",
+      "length": [
+        1,
+        2
+      ],
+      "required": true
+    },
+    "patient_name": {
+      "labels": {
+        "tiny": {
+          "en": "patient_name"
+        },
+        "description": {
+          "en": "Patient name"
+        },
+        "short": {
+          "en": "Patient name"
+        }
+      },
+      "position": 3,
+      "type": "string",
+      "length": [
+        3,
+        30
+      ],
+      "required": true
+    }
+  }
+}
+```
+
+{{% alert title="Note" %}} Users will register a new person by sending a text message in the format `N <household id> <gender> <age> <name>`. For example, `N 12345 F 20 Jane Cho` will register `Jane Cho` who belongs to the household with household ID `12345`. {{% /alert %}}
+
+To set the validation rules and autoresponses, edit the array corresponding to the `registration` key in `app_settings.json`. Add the following object within the array.
+
+```json
+{
+  "form": "N",
+  "events": [
+    {
+      "name": "on_create",
+      "trigger": "add_patient",
+      "params": "{ \"parent_id\": \"place_id\" }",
+      "bool_expr": ""
+    }
+  ],
+  "validations": {
+    "join_responses": false,
+    "list": [
+      {
+        "property": "place_id",
+        "rule": "regex('^[0-9]{5,13}$')",
+        "translation_key": "messages.validation.household_id"
+      },
+      {
+        "property": "gender",
+        "rule": "iEquals('m') || iEquals('f')",
+        "translation_key": "messages.n.validation.gender"
+      },
+      {
+        "property": "age",
+        "rule": "between(5,130)",
+        "translation_key": "messages.n.validation.age"
+      }
+    ]
+  },
+  "messages": [
+    {
+      "event": "report_accepted",
+      "translation_key": "messages.n.report_accepted",
+      "recipient": "reporting_unit"
+    }
+  ]
+}
+```
+
+{{% alert title="Note" %}} You can also define your own [validation rules]({{< relref "apps/reference/app-settings#validations" >}}).  {{% /alert %}}
+
+{{% alert title="Note" %}} `translation_key` represents the message that is sent out. This will be defined in a translations file (covered in the translations tutorial). {{% /alert %}}
```

Comment:
Another double notes to avoid if possible.

Reply:
Updated.